modules/ipi-install-replacing-a-bare-metal-control-plane-node.adoc

```diff
@@ -0,0 +1,156 @@
+// This is included in the following assemblies:
+//
+// installing/installing_bare_metal_ipi/ipi-install-expanding-the-cluster.adoc
+
+:_content-type: PROCEDURE
+[id='replacing-a-bare-metal-control-plane-node_{context}']
+= Replacing a bare metal control plane node
+
+Use the following procedure to replace an installer-provisioned {product-title} cluster.
+
+[IMPORTANT]
+====
+When the replacement BareMetalHost is created from a new physical machine or an incomplete prior YAML file configuration, the `externallyProvisioned:` field in the configuration file must be set to `false` to allow the cluster to provision the host. 
```

Comment:
Thanks, Katie. 
Regarding externallyProvisioned: field, I think the issue in the customer case had been that it was set to true, but things should work smoothly even if it isn't set at all (meaning, it doesn't have to be set to 'false' explicitly). 
Either way, I don't think that sentence is very clear. Maybe 'previous, incomplete yaml file configuration' or something else that will make the meaning more clear?

Reply:
I agree it's not very clear. Adina is also correct that it's fine to leave the field unset, since `false` is the default.

A straightforward way to say this would be:

> If you reuse or copy-and-paste the BareMetalHost from an existing control plane host, do not leave the `externallyProvisioned` field set to `true`.

That might be, uh, more straightforward than we're going for, idk.

Reply:
Updated as suggested.